gnome/libbonoboui/Portfile

```
@@ -43,7 +44,7 @@ depends_lib         port:desktop-file-utils \
                     port:libffi \
                     port:libgnome \
                     port:libgnomecanvas \
-                    port:pangox-compat
+                    port:pango
```

when changing a `depends_lib` dependency you'll need to increase the `revision`

---

Thanks, that's right, I had forgotten to do it.

---

Any port that depends on pango should do so using `path:lib/pkgconfig/pango.pc:pango` so that pango-devel can satisfy it.
```suggestion
                    path:lib/pkgconfig/pango.pc:pango
```